src/main/java/core/basesyntax/service/impl/ProcessDataServiceImpl.java

```diff
@@ -4,17 +4,16 @@
 import core.basesyntax.model.Transaction;
 import core.basesyntax.service.ProcessDataService;
 import core.basesyntax.strategy.OperationStrategy;
-import core.basesyntax.strategy.OperationStrategyImpl;
 import core.basesyntax.strategy.handler.OperationHandler;
 import java.util.List;
 import java.util.Map;
 
 public class ProcessDataServiceImpl implements ProcessDataService {
-
     @Override
     public void processData(List<Transaction> transactions,
-                            Map<Operation, OperationHandler> operationsMap) {
-        OperationStrategy strategy = new OperationStrategyImpl(operationsMap);
+                            Map<Operation, OperationHandler> operationsMap,
+                            OperationStrategy strategy) {
```

Comment:
again:
let's pass strategy in the `constructor` of ProcessDataServiceImpl
and you don't need to pass map anymore

Reply:
hmm, hopefully i understant u correct if in this case i just shoud only remove map , cause strategy i alredy added ?